menu/menu.yaml

```diff
@@ -42,14 +42,6 @@
       Title: "Monitoring NServiceBus Demo - Message backlogs"
     - Url: tutorials/monitoring-demo/walkthrough-3
       Title: "Monitoring NServiceBus Demo - Struggling endpoints"
-  - Title: More tutorials
-    Articles:
-    - Url: tutorials/message-replay
```

Comment:
The folder is still there, are we linking it from other places or should it be deleted?